React/tic-tac/src/app/screens/Login/components/formField/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import style from '../formField/style.scss';
```

Comment:
Is this not in the same folder as the `index.js` file?  If so, you can use `./style.scss` instead! 😺